app/views/lessons/show.html.haml

```diff
@@ -1,21 +1,42 @@
 .container
   .row
-    .col.s12.center
-      %h1= @lesson.title
-      %fr-view= @lesson.content.html_safe
+    .col.s12
+      %h1.center= @lesson.title
+      = @lesson.content.html_safe
 
-      - if user_signed_in? && !@enrollment.nil? && @enrollment.leader?
-        %col.s4= link_to 'Delete Lesson', course_lesson_path(@course, @lesson), method: :delete
-        %col.s4= link_to 'Update Lesson', edit_course_lesson_path(@course, @lesson)
-        %col.s4= link_to 'Create Question', new_question_path(lesson_id: @lesson.id)
+      - if @enrollment.leader?
+        .row.center
+          %a.btn-floating.waves-effect.waves-light.teal#delete-lesson{ href: course_lesson_path(@course, @lesson), "data-method" => "delete" }
+            %i.material-icons remove
+          %a.btn-floating.waves-effect.waves-light.teal#edit-lesson{ href: edit_course_lesson_path(@course, @lesson) }
+            %i.material-icons mode_edit
 
-    .container
-      .row.center
-        .col.s12
-        %h5 Assignments
-        - @questions.each do |question|
-          %ul
-            %li= link_to question.question ,questions_path(lesson_id: @lesson.id)
+    .section
+      .container
+        .row.center
+          .col.s12
+            %h5 Questions
+            - @questions.each do |question|
+              %ul.card-panel.hoverable
+                .row.valign-wrapper
+                  - if @enrollment.leader?
+                    .col.s10.valign
+                      %li= link_to question.question, question_answers_path(question)
+                    .col.s1.valign
+                      %a.btn-floating.waves-effect.waves-light.teal.delete-question{ href: question_path(question, lesson_id: @lesson.id), "data-method" => "delete"}
```

Comment:
Hash attribute should end with one space before the closing brace<br>Line is too long. [165/80]